.circleci/config.yml

```diff
@@ -1,18 +1,20 @@
 version: 2.1
 
 commands:
-  checkout_and_install:
+  npm_install:
     description: Check out the repo and install the node dependencies
     steps:
-      - checkout
+      - run:
+          name: Hash package lock
+          command: ./.circleci/scripts/hash-package-lock.sh
       - restore_cache:
           name: Load node_modules from the cache if they haven't changed
           keys:
-            - v1-npm-deps-{{ checksum "package-lock.json" }}
+            - v1-npm-deps-{{ checksum "package-lock.json.md5" }}
             - v1-npm-deps-
       - run:
           name: Install node dependencies
-          command: npm ci
+          command: npm i
```

Comment:
Just curious what was wrong with `npm ci`?

Reply:
The main problem is that `npm ci` will never use a cache:

> If a `node_modules` is already present, it will be automatically removed before `npm ci` begins its install.

https://docs.npmjs.com/cli/v6/commands/npm-ci